src/api/app/models/notification/rss_feed_item.rb

```diff
@@ -1,44 +1,53 @@
-class Notifications::RssFeedItem < Notifications::Base
+class Notification::RssFeedItem < Notification
   MAX_ITEMS_PER_USER = 10
   MAX_ITEMS_PER_GROUP = 10
 
   def self.cleanup
     User.all_without_nobody.find_in_batches batch_size: 500 do |batch|
       batch.each do |user|
-        if user.is_active?
-          ids = user.rss_feed_items.pluck(:id).slice(MAX_ITEMS_PER_USER..-1)
-          user.rss_feed_items.where(id: ids).delete_all
-        else
-          user.rss_feed_items.delete_all
-        end
+        offset = user.is_active? ? MAX_ITEMS_PER_USER : 0
+        ids = user.rss_feed_items.offset(offset).pluck(:id)
+        user.rss_feed_items.where(id: ids).delete_all
       end
     end
     Group.find_in_batches batch_size: 500 do |batch|
       batch.each do |group|
-        ids = group.rss_feed_items.pluck(:id).slice(MAX_ITEMS_PER_GROUP..-1)
+        ids = group.rss_feed_items.offset(MAX_ITEMS_PER_GROUP).pluck(:id)
         group.rss_feed_items.where(id: ids).delete_all
       end
     end
   end
+
+  def title
+    event.subject
+  end
+
+  def description
+    ApplicationController.renderer.new.render(
+      template: "event_mailer/#{event.template_name}",
+      layout: false,
+      format: :txt,
+      assigns: { host: ::Configuration.obs_url, configuration: ::Configuration.first },
+      locals: { event: event})
```

Comment:
Well it might be the test that makes this 'work'. It basically tests that 2 identical `ApplicationController.renderer` calls do the same...